app/helpers/vacancies_helper.rb

```diff
@@ -16,7 +16,7 @@ def uncapitalize_words(location_name)
 
   def page_title_prefix(vacancy, form_object, page_heading)
     if vacancy.published?
-      "#{form_object.errors.present? ? 'Error: ' : ''}Edit the #{page_heading}"
+      "#{form_object.errors.present? ? 'Error: ' : ''}Edit the #{page_heading} for #{vacancy.job_title}"
```

Comment:
This would be a bit more readable I think if it was written like this:

`"#{'Error: ' if form_object.errors.present?}Edit the #{page_heading} for #{vacancy.job_title}"`

Reply:
i think @cesidio  there are a bunch of changes (well pages missing the 'error' page title state) that needs to be addressed that i have on my list of tickets to create today, so ill leave this as is and mention this in ticket (i didnt think to improve code readability here just added what i added)